src/renderer/src/routes/+page.svelte

```diff
@@ -31,6 +31,7 @@
 			}
 
 			if (setupData.isSuccess) {
+				window.sessionStorage.setItem('keystoreInitialized', 'true');
```

Comment:
left a similar comment in another instance of `window.sessionStorage` usage. Does this get cleaned up?

Reply:
Yes, [sessionStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) lives for the duration of a browser session so it will be cleared after a restart of launcher which I think serves the purpose here.